schema/rdl_schema_0.1.json

```diff
@@ -4,123 +4,130 @@
-							"description": "Who added the resource"
-						},
-						"maintainer_email": {
-							"type": "string",
-							"description": "Contact email"
-						}
-					}
-				},
-				"resources": {
-					"type": "array",
-					"items": {
-						"resource": {
-							"type": "object",
-							"required": ["name", "format", "epsg", "url"],
-							"properties": {
-								"name": {
-									"type": "string",
-									"description": "Name of linked resource"
-								},
-								"aggregation_type": {
-									"type": "string",
-									"description": "Level of data representation"
-								},
-								"format": {
-									"type": "string",
-									"description": "Format of resource"
-								},
-								"h-res": {
-									"type": "string",
-									"description": "Horizontal resolution in meters (raster data only)"
-								},
-								"epsg": {
-									"type": "string",
-									"description": "Reference coordinate system (default: 4326)"
-								},
-								"url": {
-									"type": "string",
-									"description": "URL of linked resource"
-								}
-							}
-						}
+		"title": {
+			"type": "string",
+			"description": "Title of the dataset"
+		},
+		"abstract": {
+			"type": "string",
+			"description": "Short description of the resource"
+		},
+		"component": {
+			"type": "string",
+			"description": "Univocal schema for this contribution (H, E, V or L)",
+			"enum": [
+				"Hazard",
+				"Exposure",
+				"Vulnerability",
+				"Loss"
+			]
+		},
+		"organization": {
+			"type": "string",
+			"description": "Name of organization that published the dataset"
+		},
+		"model_source": {
+			"type": "string",
+			"description": "Name of source model"
+		},
+		"model_date": {
+			"type": "string",
+			"description": "Model release date",
+			"format": "date"
+		},
+		"version": {
+			"type": "integer",
+			"description": "Version of the dataset"
+		},
+		"purpose": {
+			"type": "string",
+			"description": "Purpose for what the data has been produced"
+		},
+		"project": {
+			"type": "string",
+			"description": "Project under which data has been produced"
+		},
+		"notes": {
+			"type": "string",
+			"description": "Additional details about the dataset"
+		},
+		"biblio_title": {
+			"type": "string",
+			"description": "Authors/year/title of publication"
+		},
+		"biblio_url": {
+			"type": "string",
+			"description": "URL of publication",
+			"format": "uri"
+		},
+		"geo_coverage": {
+			"$ref": "#/definitions/common_iso",
+			"description": "ISO code(s) of countries covered by the dataset, comma-separated"
+		},
+		"publish": {
+			"type": "boolean",
+			"description": "Flag to show/hide dataset from website"
+		},
+		"license_code": {
+			"$ref": "#/definitions/common_license",
+			"description": "Type of license"
+		},
+		"maintainer": {
+			"type": "string",
+			"description": "Who added the resource"
+		},
+		"maintainer_email": {
+			"type": "string",
+			"description": "Contact email"
+		}
+	},
+	"resources": {
```

Comment:
`resources` should be under `properties`.

Reply:
Fixed in https://github.com/GFDRR/rdl-standard/pull/93/commits/60cd76ca7bc2205f3de29177d018ea9e4ca5580c